sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeActionsOptions.java

```diff
@@ -10,8 +10,30 @@
  */
 @Fluent
 public final class AnalyzeActionsOptions {
+    private String autoDetectionDefaultLanguage;
```

Comment:
I don't recall this name being discussed in archboard, was it suggested by the Java architect?

Reply:
The named adopt from .NET API naming.https://apiview.dev/Assemblies/Review/ecd312dc5c5e42ee80e0bf9e0b6cc80d#Azure.AI.TextAnalytics.SingleLabelClassifyOptions